ember-headless-form/src/components/headless-form.hbs

```diff
@@ -1,12 +1,14 @@
-{{! ignoring prettier here is need to *not* wrap the modifier usage below into a new line, making @glint-expect-error fail to work 🙈 }}
-{{! prettier-ignore }}
 <form
   ...attributes
   {{on 'submit' this.onSubmit}}
-  {{! @glint-expect-error: modifier helper not supported, see https://github.com/typed-ember/glint/issues/410 }}
-  {{(if this.fieldValidationEvent (modifier this.on this.fieldValidationEvent this.handleFieldValidation))}}
-  {{! @glint-expect-error: modifier helper not supported, see https://github.com/typed-ember/glint/issues/410 }}
-  {{(if this.fieldRevalidationEvent (modifier this.on this.fieldRevalidationEvent this.handleFieldRevalidation))}}
+  {{(if
+    this.fieldValidationEvent
```

Comment:
This error goes away with `modifier: any`?

Reply:
Yeah, Glint does not know about the `modifier` helper (not modifiers in general! they work fine), and `any` basically turns off all type checking by saying "this thing can act as _anything_".

Reply:
is there an open issue about that? seems like it should know about `modifier` 🙃

Reply:
ah, right here: https://github.com/typed-ember/glint/issues/410